<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface_Assiduite_RH</class>
 <widget class="QMainWindow" name="Interface_Assiduite_RH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion d'assiduite</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>460</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Traiter les données</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_assiduite_ajouter">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_assid_id">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_assiduite_verifier">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_assiduite_id">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>74</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Assiduité Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_travailleur_id">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Travailleur Id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_assiduit_modifier">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>340</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_assiduite_supp">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>340</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>600</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Assiduité de travailleur</string>
      </property>
      <widget class="QLabel" name="label_date_depart">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>70</y>
         <width>143</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Date de depart en congé</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_assi_date_depart">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_date_reprise">
       <property name="geometry">
        <rect>
         <x>348</x>
         <y>70</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Date de reprise</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_assi_date_reprise">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_type_conge">
       <property name="geometry">
        <rect>
         <x>348</x>
         <y>30</y>
         <width>87</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Type de congé</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nb_absence">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>122</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre d'absence</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nb_retard">
       <property name="geometry">
        <rect>
         <x>348</x>
         <y>110</y>
         <width>105</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre de retard</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nb_hs">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>198</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre d'heures supplémentaires</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_assi_hs">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>150</y>
         <width>42</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_assi_type_conge">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>30</y>
         <width>112</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_assi_absence">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>42</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_assi_retard">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>110</y>
         <width>42</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_periode">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>46</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Période</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_assi_periode">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_presence">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>150</y>
         <width>58</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Présence</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_assi_presence">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>150</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Present(e)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Absent(e)</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QFrame" name="logo_edf">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>169</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/EdfErp.png);
image: url(:/images/logo_edf_erp.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_id_emp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Afficher les données</string>
     </attribute>
     <widget class="QTableView" name="tableView_assiduite">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_assiduite_actualiser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_assid_id</tabstop>
  <tabstop>lineEdit_assi_periode</tabstop>
  <tabstop>lineEdit_assi_date_depart</tabstop>
  <tabstop>spinBox_assi_absence</tabstop>
  <tabstop>spinBox_assi_hs</tabstop>
  <tabstop>lineEdit_assi_type_conge</tabstop>
  <tabstop>lineEdit_assi_date_reprise</tabstop>
  <tabstop>spinBox_assi_retard</tabstop>
  <tabstop>pushButton_assiduite_ajouter</tabstop>
  <tabstop>pushButton_assiduit_modifier</tabstop>
  <tabstop>pushButton_assiduite_supp</tabstop>
  <tabstop>pushButton_assiduite_verifier</tabstop>
 </tabstops>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
